### PR TITLE
testing cleanup

### PR DESCRIPTION
**Pull request by zadunn**

### CODE REVIEW

File: test/integration/default/serverspec/kafka_spec.rb [hunk 1]
```diff
@@ -55,50 +55,44 @@
...
   it 'should own all files' do
     # Ensure we reload ruby's usernames/groups
     Etc.endgrent
     Etc.endpwent
-    Dir["/opt/kafka/**/*"].each do |filePath|
-      expect(Etc.getpwuid(File.stat(filePath).uid).name).to eq("kafka")
-      expect(Etc.getgrgid(File.stat(filePath).gid).name).to eq("kafka")
+    Dir['/opt/kafka/**/*'].each do |filePath|
+      expect(Etc.getpwuid(File.stat(filePath).uid).name).to eq('kafka')
+      expect(Etc.getgrgid(File.stat(filePath).gid).name).to eq('kafka')
```
Comment (smedefind) [inline test/integration/default/serverspec/kafka_spec.rb]: Any reason not to use the existing ServerSpec file resources for this?

File: test/integration/default/serverspec/kafka_offset_monitor_spec.rb [hunk 2]
```diff
@@ -17,42 +17,42 @@
     # Ensure we reload ruby's usernames/groups
     Etc.endgrent
     Etc.endpwent
-    Dir["/opt/kafka-offset-monitor/**/*"].each do |filePath|
-      expect(Etc.getpwuid(File.stat(filePath).uid).name).to eq("kafka")
-      expect(Etc.getgrgid(File.stat(filePath).gid).name).to eq("kafka")
+    Dir['/opt/kafka-offset-monitor/**/*'].each do |filePath|
+      expect(Etc.getpwuid(File.stat(filePath).uid).name).to eq('kafka')
+      expect(Etc.getgrgid(File.stat(filePath).gid).name).to eq('kafka')
```
Comment (smedefind) [inline test/integration/default/serverspec/kafka_offset_monitor_spec.rb]: Any reason not to use the existing ServerSpec file resources for this?

File: test/integration/default/serverspec/kafka_offset_monitor_spec.rb [hunk 3]
```diff
@@ -17,42 +17,42 @@
...
-    consumerOutput = `/opt/kafka/bin/kafka-console-consumer.sh --zookeeper localhost:2181 --group #{groupName} --whitelist #{topicName} --consumer-timeout-ms 500 2>&1`
-    expect(consumerOutput).to include("Consumed 0 messages")
+    consumer_output = `/opt/kafka/bin/kafka-console-consumer.sh --zookeeper localhost:2181 --group #{group_name} --whitelist #{topic_name} --consumer-timeout-ms 500 2>&1`
+    expect(consumer_output).to include('Consumed 0 messages')
 
-    wgetOutput = `wget http://localhost:8080/#/group/#{groupName} 2>&1 | grep response`
-    expect(wgetOutput).to include("200 OK")
+    wget_output = `wget http://localhost:8080/#/group/#{group_name} 2>&1 | grep response`
+    expect(wget_output).to include('200 OK')
   end
```
Comment (smedefind) [inline test/integration/default/serverspec/kafka_offset_monitor_spec.rb:47]: Any reason not to use command resource?

File: test/integration/default/serverspec/kafka_spec.rb (hunk 1 above)
Comment (zadunn) [inline test/integration/default/serverspec/kafka_spec.rb]: Cuz copy and paste?  Also, can the file resource take a glob? e.g. could this read:

```
describe file('/opt/kafka/**/*'') do
  it { should be_owned.by('kafka') }
  it { should be_grouped.by('kafka') }
  it { should be_readable.by_user('kafka') }
  it { should be_writeable.by_user('kafka') }
end

```

**Pull request merged**